src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -227,6 +227,25 @@ def test_regiontrans_wrong_options():
     assert ("Transformation apply method options argument must be a "
             "dictionary but found 'str'." in str(excinfo.value))
 
+
+def test_parallellooptrans_refuse_codeblock():
```

Comment:
As you have also updated ParallelRegionTrans, could you please add an associated test for this too.

Reply:
Oh yes, done.